_config.php

```diff
@@ -1,14 +0,0 @@
-<?php
```

Comment:
Not having a `_config.php` file at all might cause problems. From memory there's a bit in framework that won't recognise a packages is a silverstripe module if that file doesn't exist?

Just leave an empty `_config.php` file in place with:
```php
<?php

```

Reply:
DONE. I restored file and left message.

Reply:
The problem wasn't removing the deprecation messages. The problem was that we need a `_config.php` file  in the root of the module ... even if it's completely empty.

Reply:
e.g.: https://github.com/silverstripe/silverstripe-blog/blob/3/_config.php

Reply:
DONE